source/common/http/conn_pool_grid.cc

```diff
@@ -327,7 +328,11 @@ ConnectionPool::Cancellable* ConnectivityGrid::newStream(Http::ResponseDecoder&
   }
   if (!delay_tcp_attempt) {
     // Immediately start TCP attempt if HTTP/3 failed recently.
-    wrapped_callbacks_.front()->tryAnotherConnection();
+    auto try_and_status = wrapped_callbacks_.front()->tryAnotherConnection();
```

Comment:
nit: Using destructuring `auto [try, status] = ...` can be quite nice here.

Reply:
ooh good call (if deprecated by the refactor). I'm not used to the options 17 brings!